pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -2170,7 +2239,7 @@ class Build {
                                 }
                             }
                         }
-
+                        runUploadTestImage()
```

Comment:
Should this be moved into the `if`?

Reply:
if block is inside runUploadTestImage(). See https://github.com/ibmruntimes/ci-jenkins-pipelines/blob/9dde6e98c6152c798d61a5b331517de2c3721d65/pipelines/build/common/openjdk_build_pipeline.groovy#L532

Reply:
I don't think I was clear. Should we move this 2 lines down inside the ` if (buildConfig.TEST_LIST.size() > 0)` so that we don't need to go to that function if there are no targets.

Reply:
yes, thanks. The PR is updated.